use helm for installing cert-manager

semik opened the pull request:
close #22

semik committed:
use helm for installing cert-manager

jakub-moravek reviewed (changes requested):
Move URL to variable, so  it is possible to modify it when executing playbook. In general all values like URLs, directories, .,.. should be in variables.

semik committed:
make version of cert-manager configurable

according https://docs.ansible.com/ansible/2.9/user_guide/playbooks_reuse_roles.html#role-default-variables
variables defined inside role/defaults have lowest priority. This looks
prety usefull for seting latest tested version but still allow user to
easy override our default value.

semik committed:
adding also URL as requested

semik commented:
I think that most valuable is version not the url of cert-manager repository. So added version, repo name and repo url.
